lib/util.js

```diff
@@ -1,7 +1,9 @@
 'use strict';
 
 exports.getValueAsString = function getValueString(value) {
-	if (Number.isNaN(value)) {
+	if (typeof value === 'bigint') {
+		return `${value}`;
```

Comment:
Is the `n` suffix valid for both openmetrics and prom?

Reply:
In this case, the string representation of BigInt does not include the n suffix.